### PR TITLE
SNAP-1055 Last Middle and First Name Query

### PR DESCRIPTION
**Pull request by ggomez-cambriasolutions**
### Jira Story

https://osi-cwds.atlassian.net/browse/SNAP-1055

## Description
This adds a query builder for the last, middle, and first name search combination.

## Tests
- [x] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: spec/support/helpers/query_builder_helper.rb [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
 module QueryBuilderHelper
+  def size
```
Comment (pokharelmanish) [inline spec/support/helpers/query_builder_helper.rb]: why method?? we can simply make a constant. am I missing something?

File: spec/support/helpers/query_builder_helper.rb [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+
+  def track_scores
+    'true'
+  end
+
+  def sort
+    [{ "_score": 'desc', "last_name.keyword": 'asc', "first_name.keyword": 'asc', "_uid": 'desc' }]
+  end
+
+  def min_score
```
Comment (pokharelmanish) [inline spec/support/helpers/query_builder_helper.rb]: same

File: spec/support/helpers/query_builder_helper.rb [hunk 3]
```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
 module QueryBuilderHelper
+  def size
+    '25'
+  end
+
+  def track_scores
```
Comment (pokharelmanish) [inline spec/support/helpers/query_builder_helper.rb]: same

File: spec/support/helpers/query_builder_helper.rb (hunk 1 above)
Comment (ggomez-cambriasolutions) [reply spec/support/helpers/query_builder_helper.rb]: good call. I'll switch them.

Comment (ggomez-cambriasolutions) [reply spec/support/helpers/query_builder_helper.rb]: I was going to make them accept parameters, to set the values for the tests. But it's unnecessary, so I'll make them constants.

**Pull request merged**